play/src/components/GameSessionSwitch.tsx

```diff
@@ -97,9 +99,29 @@ export default function GameSessionSwitch({
           localModel={localModel}
           currentQuestionIndex={gameSession.currentQuestionIndex}
           isShortAnswerEnabled={isShortAnswerEnabled}
+          gameSession={gameSession}
+          newPoints={0}
         />
       );
     case GameSessionState.CHOOSE_TRICKIEST_ANSWER:
+      return (
+        <GameInProgress
+          {...gameSession}
+          apiClients={apiClients}
+          teamMemberAnswersId={localModel.teamMemberAnswersId}
+          teamAvatar={localModel.selectedAvatar}
+          answerChoices={answerChoices}
+          teamId={localModel.teamId}
+          score={score}
+          hasRejoined={hasRejoined}
+          currentTimer={currentTimer}
+          localModel={localModel}
+          currentQuestionIndex={gameSession.currentQuestionIndex}
+          isShortAnswerEnabled={isShortAnswerEnabled}
+          gameSession={gameSession}
+          newPoints={0}
```

Comment:
can we set newPoints as an optional parameter in the GameInProgress interface and then just not pass it for these components? Cleaner than just passing 0 in

Reply:
Similar to how you did the `stars` parameter below

Reply:
done